ldms/src/contrib/sampler/variorum_sampler/variorum_sampler.c

```diff
@@ -0,0 +1,234 @@
+#define _GNU_SOURCE
```

Comment:
We need a copyright statement at the top of the every source code file. I recommend the following comments at the top of this file:
```
/* -*- c-basic-offset: 8 -*- */
/* Copyright 2022 Lawrence Livermore National Security, LLC
 * See the top-level COPYING file for details.
 *
 * SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
 */
```